cost/aws/instance_cloudwatch_utilization/README.md

```diff
@@ -0,0 +1,28 @@
+## AWS Instance CloudWatch Utilization Policy
+
+### What it does
+
+This Policy Template checks aws regions configured in RightScale and gets CloudWatch data for instances
```

Comment:
Should include a note about the timeframe used to collect metrics and our recommendation on how often to run the policy based on that timeframe.